### PR TITLE
refactor: Replace IIIFServiceManager (DEV-1799)

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

Issue Number: DEV-1799

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]

**Comment by seakayone** [comment]
I had to remove the `orDie` from the `IIIFService` implementation as it lead to the `Fiber` being canceled and thus the function `org.knora.webapi.responders.v2.ResourceUtilV2Live#doSipiPostUpdate` never produced any result which lead to the test failing.

File: webapi/src/main/scala/org/knora/webapi/core/AppServer.scala [hunk 1]
```diff
@@ -104,6 +104,7 @@ final case class AppServer(
                  ZIO.logError("IIIF service not running") *> ZIO.die(new Exception("IIIF service not running"))
              }
              .when(requiresIIIFService)
+             .orDie
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/core/AppServer.scala:107]: this is of course a discussion for another PR... but should we really die without retry first?

**Review by BalduinLandolt — APPROVED** [review]
Still convinced :)

**Review by mpro7 — APPROVED** [review]

File: webapi/src/main/scala/org/knora/webapi/core/AppServer.scala (hunk 1 above)
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/core/AppServer.scala:107]: IMHO dying during the startup is acceptable for now.

**Pull request merged**